docs/source/export.rst

```diff
@@ -1,11 +1,293 @@
+.. _torch.export:
+
 torch.export
 =====================
 
-.. TODO: Add torch.export() tutorial here.
-
 .. warning::
     This feature is a prototype and may have compatibility breaking changes in the future.
 
+
+Overview
+--------
+
+``torch.export`` is an ahead-of-time (AOT) process built to fully convert
+define-by-run PyTorch code into a static and standardized model representation,
+with strong soundness guarantees. It is designed to streamline and standardize the
+way PyTorch interacts with various systems and platforms.
+
+``torch.export`` creates a full graph representation of a PyTorch model
+that can be saved and loaded and run in different environments and
+languages. In comparison, ``torch.compile`` is a JIT compiler which provides
+the flexibility of creating partial graphs, where untraceable parts of models
+result in a "graph break" and will be defaulted back to the eager Python runtime.
+
+``torch.export`` maintains a clean intermediate representation (IR) with the
+following invariants. More specifications about the IR can be found here (coming
+soon!).
+
+* **Normalized**: There are no Python semantics within the graph
+
+* **Defined Operator Set**: The graph produced contains a small defined
+  :ref:`Core ATen IR <torch.compiler_ir>` opset.
+
+* **Graph properties**: The graph is purely functional, meaning it has no side
+  effects.
+
+*  **Metadata**: The graph contains metadata captured during tracing, such as a
+   stacktrace from user's code.
+
+Under the hood, ``torch.export`` leverages the following latest technologies:
+
+* **TorchDynamo (torch._dynamo)** is an internal API that uses a CPython feature
+  called the Frame Evaluation API to safely trace PyTorch graphs. This
+  provides a massively improved graph capturing experience, with much fewer
+  rewrites needed in order to fully trace the PyTorch code.
+
+* **AOT Autograd** provides a functionalized PyTorch graph and ensures the graph
+  is lowered to the small defined Core ATen operator set.
+
+* **Torch FX (torch.fx)** is the underlying representation of the graph,
+  allowing flexible Python-based transformations.
+
+
+Exporting a PyTorch Model
```

Comment:
If this doc is meant for end users of torch.export its important to also mention serialization here and how to run the exported graph